tests/unit/test_reporter.py

```diff
@@ -64,7 +64,7 @@ def test_should_write_to_default_output(
             reporter = Reporter()
             reporter.write(fake_results)
 
-            mocked__render.assert_called_once_with("html.jinja", context, False)
+            mocked__render.assert_called_once_with("report.jinja", context, False)
```

Comment:
I know that exists the `jinja` extension, but, wouldn't `report.hml `be better? Jinja resources working fine with .html extension.

Reply:
@marcuxyz changed!